src/api-next/api-next.ts

```diff
@@ -680,10 +671,10 @@ export class IBApiNext {
     subscription.next({
       all: {
         position: pos,
-        dailyPnL: undefineMax(dailyPnL),
-        unrealizedPnL: undefineMax(unrealizedPnL),
-        realizedPnL: undefineMax(realizedPnL),
-        marketValue: undefineMax(value),
+        dailyPnL: dailyPnL,
+        unrealizedPnL: unrealizedPnL,
+        realizedPnL: realizedPnL,
+        marketValue: value,
```

Comment:
did not verify it, but could be possible that you forgot to change the interface/type definition to allow undefined.

For example this.
dailyPnL? : number

Reply:
The type on IBApiNext are correct already as we did filter MAX_VALUE already, but just noticed that event callbacks in IBApi still need to be adapted:
```
on(
    event: EventName.pnl,
    listener: (
      reqId: number,
      dailyPnL: number,
      unrealizedPnL: number,
      realizedPnL: number
    ) => void
  ): this;
```
needs to become
```
on(
    event: EventName.pnl,
    listener: (
      reqId: number,
      dailyPnL?: number,
      unrealizedPnL?: number,
      realizedPnL?: number
    ) => void
  ): this;
```
will change it